pkg/api/validation/schema.go

```diff
@@ -88,6 +101,29 @@ func (s *SwaggerSchema) ValidateBytes(data []byte) error {
 	if kind == nil {
 		return fmt.Errorf("kind not set")
 	}
+	if kind.(string) == "List" {
```

Comment:
This should ideally work for any of the list types. Currently you can test for this with `strings.HasSuffix(kind.(string), "List")`. There may be a less brittle way.

@smarterclayton & @bgrant0607  note that this is not running on our regular objects, it's running on the output of unmarshalling into an interface{}-- map[string]interface{}